_posts/2016-12-22-Rust-1.14.md

```diff
@@ -0,0 +1,356 @@
+> channels, enabling you to easily switch between stable, beta, and nightly
+> compilers and keep them updated. It makes cross-compiling simpler with binary
+> builds of the standard library for common platforms. And it runs on all
+> platforms Rust supports, including Windows.
+
+[rustup]: https://internals.rust-lang.org/t/beta-testing-rustup-rs/3316/203
+
+We had [a previous post about Rustup][prev] back in May. You can learn more
+about it there, or by checking it out [on
+GitHub](https://github.com/rust-lang-nursery/rustup.rs)
+
+[prev]: https://blog.rust-lang.org/2016/05/13/rustup.html
+
+Another exciting feature is [experimental support for WebAssembly][wasm] as a
+target, `wasm32-unknown-emscripten`. It is still early days, and there's a lot
+of bugs to shake out, so please give it a try and report them! To give you a
+small taste of how it works, one you have [emscripten] installed, compiling
+some Rust code to WebAssembly is as easy as:
+
+```bash
+$ rustup target add wasm32-unknown-emscripten
+$ echo 'fn main() { println!("Hello, Emscripten!"); }' > hello.rs
+$ rustc --target=wasm32-unknown-emscripten hello.rs
+$ node hello.js
+```
+
+[wasm]: https://users.rust-lang.org/t/compiling-to-the-web-with-rust-and-emscripten/7627
+[emscripten]: http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html
+
+The community has been doing intertesting, experimental work in this area: see
+[Jan-Erik's slides] for the workshop he ran at [Rust Belt Rust] for some
+examples, or check out [Tim's example of the classic TodoMVC project][todomvc].
+This implementation builds off of his [webplatform
+crate](https://crates.io/crates/webplatform), which exposes the DOM to Rust.
+
+[Jan-Erik's slides]: http://www.hellorust.com/emscripten/
+[Rust Belt Rust]: http://www.rust-belt-rust.com/sessions/
+[todomvc]: http://timryan.org/rust-todomvc/
+
+Speaking of platforms, a large number of platforms have gained additional
+support:
+
+For `rustc`:
+
+* `mips-unknown-linux-gnu`
+* `mipsel-unknown-linux-gnu`
+* `mips64-unknown-linux-gnuabi64`
+* `mips64el-unknown-linux-gnuabi64`
+* `powerpc-unknown-linux-gnu`
+* `powerpc64-unknown-linux-gnu`
+* `powerpc64le-unknown-linux-gnu`
+* `s390x-unknown-linux-gnu`
+
+And for `std`:
+
+* `arm-unknown-linux-musleabi`
+* `arm-unknown-linux-musleabihf`
+* `armv7-unknown-linux-musleabihf`
+
+If you're using one of these platforms, follow the instructions on the website
+to install, or add the targets to an existing installation with
+`rustup target add`.
+
+These platforms are all 'tier 2', please see our page on [platform support] for
+more details.
+
+[platform support]: https://forge.rust-lang.org/platform-support.html
+
+The landing of MIR over the last few releases means that a [number of
+improvements to compile times] have landed, with more coming in the future.
+Exact numbers are hard, as they depend on what code you're compiling, but
```

Comment:
The "exact numbers are hard" seems disingenuous; we have a benchmark suite. If we don't have good overall numbers to report, I'd just drop this sentence.